src/log.h

```diff
@@ -24,15 +24,15 @@ static inline int log_oom(void) {
 #define assert_se(x)                                                    \
         do {                                                            \
                 if (!(x)) {                                             \
-                        log_error("%s:%d (%s): assertion failed:" #x "\n", \
-                                  __FILE__, __LINE__, __PRETTY_FUNCTION__); \
+                        log_error("%s:%d (%s): assertion failed: %s",   \
+                                  __FILE__, __LINE__, __PRETTY_FUNCTION__, #x); \
```

Comment:
If that ever happened, we'd get an error from gcc, so this isn't really necessary, but OK.

Reply:
yeah, but i ran into it ;-)